CHANGELOG.md

```diff
@@ -2,7 +2,18 @@
 
 All notable changes to this project will be documented in this file.
 
-## Unreleased - as of Sprint 56 end 2017-02-27
+## Unreleased - as of Sprint 57 end 2017-03-27
+
+### Added
```

Comment:
@mfeifer This looks good.  Just need to add PR https://github.com/ManageIQ/manageiq-content/pull/76.